### ISSUE TITLE
can not answer calls with my power button nor read text one by one

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#120]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.

Good after noon, I just recently did the talkback up date from the google play 
store and I can not answer calls on my t-mobile galaxy sIII, when a caller 
calls I press the home button and nothing happens.  Also when I open up the 
texting app on my phone talkback reads my text messeges as one large group.  No 
way to click on each individual text messege.
```

Original issue reported on code.google.com by `danny.ma...@gmail.com` on 17 Aug 2012 at 4:15

**Comment by GoogleCodeExporter** [comment]
```
Answering calls with the home / power button is not a accessibility feature on 
stock Android.  The Samsung SIII implements this, but you have to enable the 
functionality in Settings > Accessibility.

We've fixed reading of grouped items in TalkBack 3.1.4 Beta #1, which is 
available from the Downloads section of the Eyes-Free site.
```

Original comment by `caseybur...@google.com` on 21 Aug 2012 at 10:05
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
thank you, have not checked if the using of my home key to answer
calls has been addressed yet.
```

Original comment by `danny.ma...@gmail.com` on 22 Aug 2012 at 12:22
